src/main/java/com/glodblock/github/client/gui/container/ContainerFluidMonitor.java

```diff
@@ -412,9 +410,7 @@ private void extractFluid(IAEFluidStack fluid, ItemStack fluidContainer, EntityP
         } else {
             partialStack.stackSize = 0;
         }
-        // Now perform the real extraction.
         actualExtract.setStackSize(toExtract);
-        this.host.getFluidInventory().extractItems(canExtract, Actionable.MODULATE, this.getActionSource());
```

Comment:
This works as a fix for dupe, but also makes it harder to fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/13805
Do you have any idea?

Reply:
I'm not familiar with how super tanks implement FluidContainer. How are locked fluids handled? I could try to insert 1mb of fake fluid before to see if it works and abort if it doesn't

Reply:
It's not only super tank but in general `FluidContainer` might refuse to store fluid. You probably can use `IFluidContainerItem#drain` with `doDrain = false`

Reply:
Using FluidUtil now to check for fluid type in the container.